src/components/Calculator.js

```diff
@@ -1,53 +1,55 @@
 /* eslint-disable react/button-has-type */
 /* eslint-disable react/self-closing-comp */
 
-import React from 'react';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
+function Calculator() {
```

Comment:
- [ ] [optional] It would have been nice to stick to the class-based component since changing to the functional-based component is a requirement in the next milestone and the review might invalidate your pr because you will have to use another branch for the next milestone, the reviewer will not be able to see the changes in the File Changed section on GitHub